zhaquirks/schneiderelectric/smoke.py

```diff
@@ -0,0 +1,174 @@
+        """Attribute definitions."""
+
+        se_led_brightness: Final = ZCLAttributeDef(
+            id=0x0000,
+            type=SELEDBrightness,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+        se_alarm_sound_level: Final = ZCLAttributeDef(
+            id=0x0001,
+            type=SEAlarmSoundLevel,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+        se_alarm_sound_mode: Final = ZCLAttributeDef(
+            id=0x0002,
+            type=SEAlarmSoundMode,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+        se_lifetime: Final = ZCLAttributeDef(
+            id=0x0003,
+            type=t.uint8_t,
+            access="r",
+            is_manufacturer_specific=True,
+        )
+        se_hush_duration: Final = ZCLAttributeDef(
+            id=0x0004,
+            type=t.uint8_t,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+        se_test_mode: Final = ZCLAttributeDef(
+            id=0x0005,
+            type=t.Bool,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+        se_silence_alarm: Final = ZCLAttributeDef(
+            id=0x0006,
+            type=t.Bool,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+        se_cluster_revision: Final = ZCLAttributeDef(
+            id=0xFFFD,
+            type=t.uint16_t,
+            access="r",
+            is_manufacturer_specific=True,
+        )
+
+
+(
+    QuirkBuilder(SE_MANUF_NAME, "W599001")
+    .applies_to(SE_MANUF_NAME, "W599501")
+    .applies_to(SE_MANUF_NAME, "755WSA")
+    .replaces(SEBasic, endpoint_id=20)
+    .replaces(SEAlarmConfiguration, endpoint_id=20)
+    .enum(
+        cluster_id=SEAlarmConfiguration.cluster_id,
+        endpoint_id=20,
+        attribute_name=SEAlarmConfiguration.AttributeDefs.se_led_brightness.name,
+        translation_key="led_brightness",
+        fallback_name="LED brightness",
+        enum_class=SELEDBrightness,
+    )
+    .enum(
+        cluster_id=SEAlarmConfiguration.cluster_id,
+        endpoint_id=20,
+        attribute_name=SEAlarmConfiguration.AttributeDefs.se_alarm_sound_level.name,
+        translation_key="alarm_sound_level",
+        fallback_name="Alarm sound level",
+        enum_class=SEAlarmSoundLevel,
+    )
+    .enum(
+        cluster_id=SEAlarmConfiguration.cluster_id,
+        endpoint_id=20,
+        attribute_name=SEAlarmConfiguration.AttributeDefs.se_alarm_sound_mode.name,
+        translation_key="alarm_sound_mode",
+        fallback_name="Alarm sound mode",
+        enum_class=SEAlarmSoundMode,
+    )
+    .sensor(
+        cluster_id=SEAlarmConfiguration.cluster_id,
+        endpoint_id=20,
+        attribute_name=SEAlarmConfiguration.AttributeDefs.se_lifetime.name,
+        translation_key="lifetime",
+        fallback_name="Lifetime",
+        device_class=NumberDeviceClass.DURATION,
```

Comment:
Per the comments above, let's remove this for now, as HA doesn't accept "y" (years) for the duration device class at the moment.
```suggestion
```